File: MDfiles/Data structures and Algorithms with Java Python and Javascript/Python/Queue.py
class Queue:

    def __init__(self):
        self.items=[]
    def enqueue(self,item):
        self.items.insert(0,item)
    def dequeue(self):
        if self.items:
            return self.items.pop()
        return None
    def size(self):
        return len(self.items)
    def is_empty(self):
        return self.items==[]
    def peek(self):
        return self.items[-1]
    def __str__(self):
        return str(self.items)

    

File: MDfiles/Data structures and Algorithms with Java Python and Javascript/Java/Queue.java
package test;

public class Queue {
	
	static final int MAX = 10;
    int top;
    int n =0;
    int a[] = new int[MAX]; // Maximum size of queue
    
    boolean isEmpty()
    {
        return (top < 0);
    }
    Queue()
    {
        top = -1;
    }
    boolean push(int x)
    {
        if (top >= (MAX - 1)) {
            System.out.println("Stack Overflow");
            return false;
        }
        else {
            a[++top] = x;
            System.out.println(x + " pushed into stack");
            return true;
        }
    }
    
    int pop()
    {
        if (top < 0) {
            System.out.println("Stack Underflow");
            return 0;
        }
        else {
            int x = a[n++];
            //--top;
            return x;
        }
    }
    
    int peek()
    {
        if (top < 0) {
            System.out.println("Stack Underflow");
            return 0;
        }
        else {
            int x = a[this.n];
            return x;
        }
    }
    
    public void showall() {
    	
    	//System.out.println("n=>"+this.top);
    	while (this.n<=this.top) {
    		System.out.print(this.a[n]+" ");
    		this.n++;
    	}
    }
 
 
    

}
